aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { ThemeContext } from 'styled-components';
+import { simpleTags } from './data';
+import {
+  alphabetize,
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from './CreateTag';
+
+const defaultValueOptions = [];
+const defaultValue = '';
+const defaultTags = [];
+
+simpleTags.forEach(datum => {
+  defaultValueOptions.push(datum);
+});
+
+export const CreateTagSimple = () => {
+  const size = useContext(ResponsiveContext);
+  const theme = useContext(ThemeContext);
+
+  // value
+  const [valueOptions, setValueOptions] = useState(
+    alphabetize(defaultValueOptions),
+  );
+  const [value, setValue] = useState(defaultValue);
+  const [valueSearch, setValueSearch] = useState('');
+
+  // tags
+  const [currentTags, setCurrentTags] = useState(defaultTags);
+
+  return (
+    <AppContainer>
+      <TagPageHeader
+        description={`Tags are values that can be assigned to 
+      resources.`}
+      />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
+              : '100%'
+          }
+          gap={{ column: 'medium', row: 'small' }}
+        >
+          <FormField
+            name="tag-value-simple"
+            htmlFor="tag-value-simple"
+            label="Value"
+            required={{ indicator: false }}
+          >
+            <Select
```

Comment:
From a UX perspective, I am wondering if a multi-select would be a better experience? That way I'd only have to open the select menu once, choose my tags, then assign. Allowing for a single value at a time is repetitive and can be somewhat frustrating.

Reply:
I could see that making for a smoother process. I remember at some point in time, we had leaned away from multi-select in Select, so I think I'd like to touch base with a designer before making this switch.

Reply:
I have filed an issue to take this topic on separately: https://github.com/grommet/hpe-design-system/issues/2207